docs/core/deploying/acquisition-experiences.md

```diff
@@ -0,0 +1,171 @@
+﻿---
+title: Acquisition experiences - .NET Core
+description: .NET Core Acquisition Experiences
+author: jamshedd
+ms.author: jamshedd
+ms.date: 09/06/2019
+---
+# .NET Core acquisition experiences
+
+There are various Acquisition options for .NET Core - 
+* [Visual Studio and Visual Studio Code](#Visual-Studio-and-Visual-Studio-Code)
+* [https://dot.net (Microsoft .NET site)](#httpsdotnet-Microsoft-NET-site)
+* [Dotnet-install.ps1/.sh Scripts](#Dotnet-install.ps1.sh-Scripts)
+* [Docker](#Docker)
+* [Linux Package Managers](#Linux-Package-Managers)
+* [Azure DevOps pipeline Installer Task](#Azure-DevOps-pipeline-Installer-Task)
+* [Snap store](#Snap-store)
+* [.NET Core on GitHub](#NET-Core-on-GitHub)
+
+Each of these choices has their own strengths. Some options support both initial acquisition, and subsequent update of the .NET Core runtime and tools, while others don't support updates.
+
+|Acquisition Choice|Acquire|Update|
+|:-|:-:|:-:|
+[Visual Studio and Visual Studio Code](#Visual-Studio-and-Visual-Studio-Code)   | ✔️ | ✔️ |
+[https://dot.net (Microsoft .NET site)](#httpsdotnet-Microsoft-NET-site)        | ✔️ | ❌ |
+[Dotnet-install.ps1/.sh Scripts](#Dotnet-install.ps1.sh-Scripts)                | ✔️ | ✔️ |
+[Docker](#Docker)                                                               | ✔️ | ✔️ |
+[Linux Package Managers](#Linux-Package-Managers)                               | ✔️ | ✔️\*|
+[Azure DevOps pipeline Installer Task](#Azure-DevOps-pipeline-Installer-Task)   | ✔️ | ✔️ |
+[Snap store](#Snap-store)                                                       | ✔️ | ✔️ |
+[.NET Core on GitHub](#NET-Core-on-GitHub)                                      | ✔️ | ❌ |
+
+_\*Some package Managers provide update notifications__
+
+Additionally, differences between these choices make some of these choices a great option for developers while others are better suited for DevOps and IT administrators.
```

Comment:
I think it would read better to avoid using `choices` multiple times.

```suggestion
Additionally, differences between these choices make some of them a great option for developers while others are better suited for DevOps and IT administrators.
```